lib/rack/attack/throttle.rb

```diff
@@ -51,6 +51,16 @@ def matched_by?(request)
           end
         end
       end
+
+      private
+
+      def discriminator_for(request)
+        discriminator = block.call(request)
+        if discriminator && Rack::Attack.discriminator_normalizer != false
```

Comment:
Consider dropping the ` != false` on end of line 59, so this will gracefully handle when discriminator_normalizer is set to `nil` or `false`.